src/julia/GF.jl

```diff
@@ -397,13 +397,17 @@ end
 #
 ###############################################################################
 
+mpz_probab_prime_p(x::Integer, reps::Integer=25) =
```

Comment:
Would this be better off in src/julia/Integer.jl?

Reply:
Your call, I put it here just as an internal function, next to the place where it's use. (I could add a leading `_` to clarify that this is an internal function).

Reply:
I think it is better to move it, to save other people implementing it "where it is needed".

Reply:
Alright, I then also renamed the function to the saner name `is_probab_prime`, and made it return `Bool` instead of `Cint`, if it's fine for you.

Reply:
There is also a Nemo function with a similar name. Maybe they could have the same name.

Reply:
Good idea, renamed to `isprobabprime`.